packages/wallets/nightly/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { PublicKey as SolanaPublicKey, Transaction as SolanaTx } from '@solana/web3.js';
+
+export declare class Nightly {
+    solana: SolanaNightly;
+    private readonly _nightlyEventsMap;
+    constructor();
+    invalidate(): void;
+}
+
+export declare class SolanaNightly {
+    publicKey: SolanaPublicKey;
+    _onDisconnect: () => void;
+    private readonly _nightlyEventsMap;
```

Comment:
Don't think you need to declare private / `_`-prefixed props and methods in the exported type?

Reply:
my bad, I've overlooked these, removed